internal/runtime/client/client.go

```diff
@@ -153,6 +163,98 @@ func (c *client) Unregister(extensionConfig *runtimev1.ExtensionConfig) error {
 	return nil
 }
 
+// CallAllExtensions calls all the ExtensionHandlers registered for the hook.
+// The ExtensionHandler are called sequentially. The function exits immediately after any of the ExtensionHandlers return an error.
```

Comment:
I'm not sure about this behavior. Is this derived/consistent with admission webhooks?

Reply:
I don't remember, but @ykakarap researched it.

But I'm pretty sure we can't be entirely consistent with how admission webhooks work today, as it's e.g. possible that mutating webhooks are called multiple times during a single admission IIRC.

Reply:
We had some discussion about always calling all of them (which can take very long if multiple are broken and timeout) or calling them in parallel, but for now we wanted to keep it as simple as possible.

Reply:
What is important for me is that we document WHY are we implementing a certain behavior, otherwise in future we will circle back to the same discussion